flocker/acceptance/testtools.py

```diff
@@ -311,9 +318,9 @@ def flocker_deploy(test_case, deployment_config, application_config):
 
     application = temp.child(b"application.yml")
     application.setContent(safe_dump(application_config))
-
-    check_call([b"flocker-deploy", control_node, deployment.path,
-                application.path])
+    check_call([b"flocker-deploy", b"--certificates-directory",
+               certificate_path, control_node, deployment.path,
```

Comment:
This will fail with an obscure error about `None` not being allowed in `check_call` argument list if `FLOCKER_ACCEPTANCE_API_CERTIFICATES_PATH` wasn't set in the environment.